presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -73,6 +73,18 @@ default ConnectorTableHandle getTableHandle(ConnectorSession session, SchemaTabl
         return null;
     }
 
+    /**
+     * Return true if table or view with specified name exists.
+     */
+    default boolean isExistingRelation(ConnectorSession session, SchemaTableName objectName)
```

Comment:
This is an SPI change now. I am not sure we want this.
@findepi, @martint ?

Reply:
I'll remove that.